module/VuFind/sql/mysql.sql

```diff
@@ -247,7 +247,9 @@ CREATE TABLE `user` (
   `last_language` varchar(30) NOT NULL DEFAULT '',
   PRIMARY KEY (`id`),
   UNIQUE KEY `username` (`username`(190)),
-  UNIQUE KEY `cat_id` (`cat_id`(190))
+  UNIQUE KEY `cat_id` (`cat_id`(190)),
+  KEY `email` (`email`),
```

Comment:
I think it would be safest to limit email key length to the same 190 characters as e.g. username. While I'm not sure if an email address with enough multi-byte characters to blow the limit is possible, I think it's safest to assume so.

Reply:
I couldn't remember the purpose of this, but it's coming back to me now. Adjusted as requested! :-)